src/components/home/partners/Partners.jsx

```diff
@@ -1,17 +1,29 @@
-import PrimaryPartner from "./PrimaryPartner";
-import SecondaryPartner from "./SecondaryPartner";
-import CallToAction from "./CallToAction";
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import PrimaryPartner from './PrimaryPartner';
+import SecondaryPartner from './SecondaryPartner';
+import CallToAction from './CallToAction';
 
 const Partners = () => {
-  // any logic needed
   return (
     <>
+      <Box>
+        <Typography
+          component={'h2'}
```

Comment:
why the different expressions? one with {'h2'} & other with "h3"

Reply:
who knows 🤪, fixed in next push for consistency.